x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/related_dashboards.tsx

```diff
@@ -37,8 +40,15 @@ export function RelatedDashboards({ alert, relatedDashboards }: RelatedDashboard
     },
   } = useKibana();
 
+  const [timeRange, setTimeRange] = useState<TimeRange>({ from: 'now-15m', to: 'now' });
+  const alertStart = alert.fields[ALERT_START];
+  const alertEnd = alert.fields[ALERT_END];
   const dashboardLocator = urlService.locators.get<DashboardLocatorParams>(DASHBOARD_APP_LOCATOR);
 
+  useEffect(() => {
+    setTimeRange(getPaddedAlertTimeRange(alertStart!, alertEnd));
```

Comment:
For some reason, it doesn't seem like the alert end date is working properly for recovered alerts. Whenever I navigate to the url, it just sets the end time as the absolute version of "now". Can you look into this further?